AndroidSDKCore/src/main/java/com/leanplum/internal/CountAggregator.java

```diff
@@ -0,0 +1,62 @@
+package com.leanplum.internal;
+
+import java.util.HashSet;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CountAggregator {
+
+    public static final CountAggregator INSTANCE = new CountAggregator();
```

Comment:
I'm not sure I like this paradigm. It's minimal and flexible _now_ but is a slippery slope. Maybe we could have a factory?